server/NewBank.java

```diff
@@ -16,19 +16,25 @@ private NewBank() {
     }
 
     private void addTestData() {
```

Comment:
We shouldn't need this test data now right? As they will be loaded in with the db... any point of these changes?